aries-site/src/data/structures/templates.js

```diff
@@ -5,6 +5,37 @@ import {
 } from '../../examples/cardPreviews';
 
 export const templates = [
+  {
+    name: 'App Layouts',
```

Comment:
So are we deleting `page layouts`

Reply:
We will still end up having a Page Layouts page, however it will be focussed on the primary page content, presenting cards, lists, tables, paragraphs, sub-sections of the page, etc. and it will ignore app level navigation and such.

Page Layouts will be more instructive as to how to work with Grid, build responsive layouts, order content based on breakpoints. It will also cover things like 1,2, and 3 column layouts.

Reply:
I see that makes sense thank you for clarifying